geoips/interfaces/module_based/output_checkers.py

```diff
@@ -517,8 +517,11 @@ def gunzip_product(fname, is_comparison_product=False, clobber=False):
     # Otherwise, see if the .gz version exists.
     elif glob(fname + ".gz"):
         gz_fname = fname + ".gz"
-    LOG.info("**** Gunzipping product for comparisons")
-    LOG.info("gunzip %s", gz_fname)
+
+    message = ["Gunzipping product for comparisons"]
+    message.append(f"gunzip {gz_fname}")
+    log_with_emphasis(LOG.info, message)
```

Comment:
This won't work currently. You'll need to edit this line to ``log_with_emphasis(LOG.info, *message)`` or update log_with_emphasis to unpack a list if provided.

Reply:
Ah 😢 thanks @evrose54 - I keep making that mistake. Thanks for catching it!

Reply:
Fixed!